20form-elements/README.md

```diff
@@ -0,0 +1,60 @@
+![phillip zastrow logo](https://www.digitalocean.com/community/tutorials/how-to-style-common-form-elements-with-css)
```

Comment:
Double check that this link is pointing to an image because it does not appear to be working for me.

Reply:
Ah, working now!
![Screen Shot 2022-08-29 at 1 03 59 PM](https://user-images.githubusercontent.com/49820744/187255129-d311d608-53c7-4109-ac09-f806d1cb75f6.png)
I accidentally posted the tutorial link there. 

This is a matter of double-checking the markup by doing the split-view in VS Code that shows you how it displays. I forgot this time, and will be more careful now that I see a display error got through!

Reply:
Here is the sha with the correction on the image URL and missing space on markup heading: 69cad37a73a0624936c94ce49be4ddd976c43ec2